build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
```

Comment:
What's our stylistic position on parens around the conditional in tertiary statements like this?